README.md

```diff
@@ -84,6 +84,12 @@ pip install --upgrade pip setuptools pipenv
 pipenv install --dev
 ```
 
+To update the design system templates run:
+
+```
+pipenv run ./scripts/load_templates.sh
```

Comment:
Can you add this to the Makefile?

Reply:
Maybe I just add it as part of the `make run` task and remove this from the readme? Seems to download and unzip the templates pretty fast

Reply:
I've updated the README and makefile now